src/process/message_processor.go

```diff
@@ -5,22 +5,12 @@ import (
 	"BGPAlert/common"
 	"BGPAlert/config"
 	"fmt"
-	"strconv"
 	"time"
 )
 
 // Constantly read messages from channel, build up windows with frequency maps, calling analysis on windows when full
 func ProcessBGPMessages(msgChannel chan common.BGPMessage, config *config.Configuration) error {
-	// Parse windowSize from config
-	var maximumBuckets int
-	// If windowSize can't be parsed to an int -> just default to 30
-	if windowSize, err := strconv.ParseInt(config.WindowSize, 10, 64); err != nil {
-		fmt.Println("Inputted config size is not a number, defaulting to 30")
-		maximumBuckets = 30
-	} else {
-		maximumBuckets = int(windowSize)
-	}
-
+	maximumBuckets := config.WindowSize
```

Comment:
to my comment above - is there ever a time that this value is checked that the value passed in is ok/acceptable?